docs/source/subscriptions.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Adding Subscriptions To Schema
+---
+
+Adding GraphQL subscriptions to your graphQL schema is simple. since Subscriptions are enabled by default in Apollo 2.0 and you dont' need to install anymore any extra packages.
+As example we want to display to display a list of post that contains author and comment (Query) and we want to add a post to them (Mutation).
+
+Our schema will be:
+
+```
+    type Query {
+        posts: [Post]
+    }
+    type Mutation {
+        addPost(author: String, comment: String): Post
+    }
+    type Post {
+        author: String
+        comment: String
+    }
+```
+
+Our resolver:
+```js
+
+const postResolver = {
+    Query: {
+        posts(root: any, args: any, context: any) {
+            return postController.posts();
+        }
+    },
+    Mutation: {
+        addPost(root: any, args: any, context: any) {
+            pubsub.publish(POST_ADDED, { postAdded: args });
+            return postController.addPost(args);
+        }
+    }
+};
+
+export default postResolver;
+
+```
+
+To enable subscription we need to apply just few changes:
+
+- In our schema:
+
+```
+    type Query {
+        posts: [Post]
+    }
+    type Mutation {
+        addPost(author: String, comment: String): Post
+    }
+    type Post {
+        author: String
+        comment: String
+    }
+    type Subscription {
+        postAdded: Post
+    }
+```
+
+We just added `type Subscription`,
+
+- In the resolver:
+```js
+const POST_ADDED = 'POST_ADDED';
+
+const postResolver = {
+    Subscription: {
+        postAdded: {
+            subscribe: () => pubsub.asyncIterator([POST_ADDED])
+        }
+    },
+    Query: {
+		posts(root: any, args: any, context: any) {
```

Comment:
The indentation on this is a bit weird. Try running the file through prettier and that will go away.  `npx prettier --write docs/source/subscriptions.md` should do the trick

Reply:
I run it but it doesn't look it does too much. If it's still not how you expect I'll have a double check